From d44e2c5461073e0d511d7ff7d373d2ffefabcb17 Mon Sep 17 00:00:00 2001
From: Jean Sainctavit <jean.sainctavit@epita.fr>
Date: Sun, 11 Dec 2022 17:47:40 +0100
Subject: [PATCH] Replace "Hello, World!" by "Hello, patched world!"

---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index 7d38eaf..c750577 100644
--- a/src/main.c
+++ b/src/main.c
@@ -2,6 +2,6 @@
 
 int main()
 {
-    printf("Hello, World!\n");
+    printf("Hello, patched world!\n");
     return 0;
 }
-- 
2.34.1

